### PR TITLE
VIH-9070 Using new JobHistory endpoint in anonymise job

### PR DESCRIPTION
**Pull request by will-craig**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-9070

### Change description ###
Implemented call to UpdateJobHistory endpoint success/failure

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Comment by will-craig** [comment]
Will need updating with stable package once bookings-api pr merged.

**Review by muralikaturi — DISMISSED** [review]

File: SchedulerJobs/SchedulerJobs.Services/JobHistoryService.cs
Comment (omio-ma) [inline SchedulerJobs/SchedulerJobs.Services/JobHistoryService.cs:8]: not sure if this project is structured like this, but optionally we could move this into its separate file, but wouldnt block the pr for that change

**Review by omio-ma — APPROVED** [review]

**Pull request merged**
